src/components/SendTokens/EthereumTokens.js

```diff
@@ -498,11 +508,19 @@ class SendEthereumTokens extends React.Component<Props, State> {
     return `${feeInEth} ETH (${formattedFeeInFiat})`;
   };
 
-  renderFeeToggle = (showFee: boolean) => {
+  renderFeeToggle = (showFee: boolean, feeError: boolean) => {
     const { txFeeInfo } = this.state;
     if (showFee && txFeeInfo) {
       const { fee, gasToken } = txFeeInfo;
-      return <FeeLabelToggle txFeeInWei={fee} gasToken={gasToken} />;
+      const gasTokenSymbol = get(gasToken, 'symbol', ETH);
+      return (
+        <>
+          <FeeLabelToggle txFeeInWei={fee} gasToken={gasToken} />
+          {!!feeError &&
+            <BaseText secondary>Not enough {gasTokenSymbol} for the transaction fee</BaseText>
```

Comment:
Let's try and be a bit nicer to our users 👵 

```suggestion
            <BaseText secondary>Sorry, you do not have enough {gasTokenSymbol} in your wallet to make this transaction. Please top up your wallet and try again.</BaseText>
```

Reply:
I'll need to see how the UI will react to these longer messages on multiple screen sizes